tests/integration/inference/test_text_inference.py

```diff
@@ -99,6 +99,30 @@ def test_text_completion_streaming(client_with_models, text_model_id, test_case)
     assert len(content_str) > 10
 
 
+@pytest.mark.parametrize(
+    "test_case",
+    [
+        "inference:completion:stop_sequence",
+    ],
+)
+def test_text_completion_stop_sequence(client_with_models, text_model_id, test_case):
+    skip_if_model_doesnt_support_completion(client_with_models, text_model_id)
```

Comment:
this test will only work with vllm right?

Reply:
There maybe others that support this but I have not tested. Submitted https://github.com/meta-llama/llama-stack/issues/1771 to track